shell/common/rasterizer_unittests.cc

```diff
@@ -111,11 +115,70 @@ TEST(RasterizerTest, drawWithExternalViewEmbedder) {
   MockExternalViewEmbedder external_view_embedder;
   EXPECT_CALL(*surface, GetExternalViewEmbedder())
       .WillRepeatedly(Return(&external_view_embedder));
+
+  auto surface_frame = std::make_unique<SurfaceFrame>(
+      nullptr, true, [](const SurfaceFrame&, SkCanvas*) { return true; });
```

Comment:
nit: add parameter name comments for parameters

Reply:
done